src/api/app/models/project.rb

```diff
@@ -766,11 +766,7 @@ def find_package(package_name, check_update_project = nil, processed = {})
   end
 
   def expand_all_repositories
```

Comment:
:thought_balloon: What about renaming this method to `expanded_repositories` to avoid the same name with the `expand_all_repositories` method below?

Reply:
Yes, kind of ugly, but I don't really want to change any public API in this PR, thank you!